addons/medical_treatment/functions/fnc_splintCondition.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: PabstMirror
+ * Can apply a splint to the patient
```

Comment:
Can you apply a splint to the head?

Reply:
limited to limbs in treatment actions.hpp
`allowedSelections[] = {"LeftArm", "RightArm", "LeftLeg", "RightLeg"};`